browser/src/layer/tile/CanvasTileLayer.js

```diff
@@ -2474,7 +2474,15 @@ L.CanvasTileLayer = L.Layer.extend({
 		}
 		this._searchTerm = originalPhrase;
 		this._map.fire('search', {originalPhrase: originalPhrase, count: count, highlightAll: highlightAll, results: results});
-		app.setFollowingUser(this._viewId, /* instant jump */ true);
+
+		app.setFollowingUser(this._viewId);
+
+		// always jump to search result - we already received cell / text cursor before so we need
+		// to force it in case we had following OFF
+		if (app.file.textCursor.visible)
+			this._onUpdateCursor(/* scroll */ true);
+		if (app.calc.cellCursorVisible)
```

Comment:
I guess we could use an else-if here.
```suggestion
		if (app.calc.cellCursorVisible)
```

Reply:
done